src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -391,6 +399,18 @@ def app_events(
         default="",
         help="Fetch events that are older than this time ago, in Snowflake interval syntax.",
     ),
+    record_types: Annotated[
+        list[RecordType], typer.Option(case_sensitive=False)
+    ] = typer.Option(
+        [],
+        "--type",
+        help="Restrict results to specific record type. Can be specified multiple times.",
+    ),
+    scopes: Annotated[list[str], typer.Option()] = typer.Option(
```

Comment:
Does this one need `Annotated`?

Reply:
according to the docs, yes: https://typer.tiangolo.com/tutorial/multiple-values/multiple-options/